src/styles/main.scss

```diff
@@ -0,0 +1,137 @@
+@import "./variables";
+@import "./reset";
+@import "./animation-stepOn60";
+@import "./extends";
+@import "./mixins";
+
+.page {
+  background: $c-bg-gradient;
+  height: 100vh;
+  margin: 0;
+
+  @extend %flex-center;
+}
+
+.watch {
```

Comment:
Move to separate file

Reply:
moved to `page.scss` dedicated to page body general styling rules

Reply:
optimised